packages/components/src/tooltip/index.js

```diff
@@ -35,13 +35,29 @@ class Tooltip extends Component {
 			TOOLTIP_DELAY
 		);
 
+		/**
```

Comment:
Should this be just a multiline comment with `//`?

Reply:
> Should this be just a multiline comment with `//`?

It's meant to be JSDoc describing the instance variable. I guess including `@type` (like in the subsequent one) could help make this clearer).